scripts/verify_ezid_status.py

```diff
@@ -0,0 +1,506 @@
+        if not id_created:
+            print(f"  Error - creating on {shoulder} failed - status_code: {status_code}: {text}: {err_msg}")
+            return
+
+        response = requests.get(f'{self.base_url}/{id_created}??')
+        if response.status_code == 200 and 'what: test record under shoulder - ark:/99999/fk4' in response.text:
+            print(f"  ok - ?? for instrospection: {id_created} returned introspection request")
+        else:
+            print(
+                f"  Error - ?? for instrospection - status_code: {response.status_code}: {response.text.strip()}")
+
+        response = requests.get(f'{self.base_url}/{id_created}?info')
+        if response.status_code == 200 and 'what: test record under shoulder - ark:/99999/fk4' in response.text:
+            print(f"  ok - ?info for instrospection: {id_created} returned introspection request")
+        else:
+            print(
+                f"  Error - ?info for instrospection - status_code: {response.status_code}: {response.text.strip()}")
+
+
+    def check_batch_download(self, notify_email):
+        print("## Check batch download from S3")
+        data = {
+            'format': 'csv',
+            'type': 'ark',
+            'column': '_id',
+            'notify': notify_email,
+        }
+        url = f"{self.base_url}/download_request"
+
+        # post download requst
+        http_success, status_code, text, err_msg = self._post_data(url, data, content_type="form")
+        if http_success and status_code == 200:
+            # download request was successfully processed with "success" and downloadable url in the response body
+            # success: https://ezid-stg.cdlib.org/s3_download/xTqyrjZDJz9zYRMz.csv.gz
+            part_1 = text[:7]
+            s3_file_url = text[9:]
+            assert part_1, "success"
+
+            # batch download is processed asynchronously, wait until the file is ready for download
+            count = 0
+            success = False
+            wait_time = 0
+            while count < 60:
+                status = self._get_status(s3_file_url, allow_redirects=True)
+                success = status['success']
+                if success:
+                    break
+                time.sleep(5)
+                wait_time += 5
+                print(f"  waiting for file to become available: {wait_time}s passed")
+                count += 1
+
+            if success:
+                print(f"  ok - batch download file is available at: {s3_file_url}")
+                print(f"  Info: Please check account {notify_email} for email with Subject: Your EZID batch download link")
+            else:
+                print(f"  Error - batch download failed: {s3_file_url} - status_code: {status_code}: {text}: {err_msg}")
+        else:
+            print(f"  Error - batch download failed - status_code: {status_code}: {text}: {err_msg}")
+
+
+    def check_resolver(self):
+        item = "Verify Resolver function"
```

Comment:
Can we add a print statement here to match output format with other testing items.

print("## Verify Resolver function")

Reply:
Yes, updated.